manager/logbroker/broker.go

```diff
@@ -383,6 +383,15 @@ func (lb *LogBroker) PublishLogs(stream api.LogBroker_PublishLogsServer) (err er
 			}
 		}
 
+		// if we have a close message, close out the subscription
+		if log.Close {
+			// Mark done and then set to nil so if we error after this point,
+			// we don't try to close again in the defer
+			currentSubscription.Done(remote.NodeID, err)
```

Comment:
Do we need to set `err` here or `nil`?

If we get here it basically means everything worked out fine right?